### DISCUSSION TITLE
LPS and serial

### DISCUSSION THREAD

**Discussion by danteppc** [bitcraze/discussions#433] [Q&A]
Hi,

I have a problem with USB connection to Loco nodes:
 
When I connect my Loco node to my Mac and `ls /dev`, I see `tty.usb*` and can do `screen` without any problem.
However, when I `make` the firmware and flash the Loco node using lpstools with the just built `/bin/lps-node-firmware.dfu` file, the node seems to function normally, however, I don't see the `tty.usb* `in my` /dev` directory anymore!

What could be the problem?

**Comment by krichardsson** [comment]
Hi,

When you set the anchor in dfu mode, it does not show up in `/dev`, but you can see it with a tool like dfu-util for instance (use `dfu-util -l`)
I'm not exactly sure what you mean, do you not see the the anchor in `/dev` after flashing and re-booting?

**Reply by danteppc** [reply]
Yes the problem is that I don't see the device in `/dev` anymore after flashing with a firmware I built. I don't think `dfu-util` can solve my problem because I need to access the config menu by doing `screen tty.usb*`.

**Reply by danteppc** [reply]
note: even when I build the current source (unmodified) and flash, the device stops appearing in `/dev`

**Reply by krichardsson** [reply]
It sounds like there might be some problem with binary you are flashing. Try building it with the [toolbelt](https://www.bitcraze.io/documentation/repository/toolbelt/master/), to see if that works. This is using the same compiler as the build servers and should create a good binary if you build from unmodified source code.
